src/app/ng-intercom/types/boot-input.ts

```diff
@@ -14,5 +14,5 @@ export interface DefaultBootInput {
     custom_launcher_selector?: string;
 }
 
-export type BootInput = ((UserWithEmail | UserWithUid) & DefaultBootInput & Any) | DefaultBootInput;
+export type BootInput = ((UserWithEmail | UserWithUid) & DefaultBootInput & Any) | DefaultBootInput
```

Comment:
Also, if you know a better way to do this than this I would love it, since this is very verbose.